enterprise/backend/src/metabase_enterprise/gsheets/settings.clj

```diff
@@ -0,0 +1,75 @@
+(ns metabase-enterprise.gsheets.settings
+  (:require
+   [metabase-enterprise.gsheets.constants :as gsheets.constants]
+   [metabase.models.setting :as setting :refer [defsetting]]
+   [metabase.premium-features.core :as premium-features]
+   [metabase.util :as u]
+   [metabase.util.i18n :refer [deferred-tru]]
+   [metabase.util.json :as json]
+   [metabase.util.malli.registry :as mr]
+   [metabase.util.malli.schema :as ms]
+   [toucan2.core :as t2]))
+
+(mr/def :gsheets/gsheets
```

Comment:
This isn't actually used, but it is good for docs

Reply:
I think the comment is good enough. I put a stub for "add malli schemas to defsetting?" to the march 20th guild sync